exercises/concept/treasure-chest/.docs/instructions.md

```diff
@@ -0,0 +1,26 @@
+# Instructions
+
+In this exercise you're going to write a generic (/ magical!) TreasureChest, to store some treasure.
+
+## 1. Define the TreasureChest generic custom type
+
+Define a `TreasureChest` custom type with a single variant.
+
+The variant should have an associated String value, which will be used to store the password of the treasure chest
+
+The variant should also have an associate generic value, which will be used to store the treasure. This value is generic so that the treasure can be anything.
```

Comment:
```suggestion
The variant should also have an associate generic value, which will be used to store the treasure.
This value is generic so that the treasure can be anything.
```

Reply:
@ceddlyburge you missed some of the comments, I think you didn't click "load more" to see them all.